### PR TITLE
docs: add update samples

### PR DESCRIPTION
**Pull request by irataxy**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [X] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-game-servers/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes 205773679

### CODE REVIEW

File: samples/test/list_clusters.test.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ describe('Game Servers List Clusters Test', () => {
     await operation2.promise();
   });
 
-  it.only('should list Game Server clusters in a realm', async () => {
+  it('should list Game Server clusters in a realm', async () => {
```
Comment (irataxy) [inline samples/test/list_clusters.test.js:82]: I can revert this change after kokoro tests pass (if needed).

File: samples/test/update_cluster_test.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    };
+
+    const [operation2] = await gameClustersClient.createGameServerCluster(
+      createClusterRequest
+    );
+    await operation2.promise();
+  });
+
+  it('should update a Game Server cluster in a realm', async () => {
+    const projectId = await realmsClient.getProjectId();
```
Comment (JustinBeckwith) [inline samples/test/update_cluster_test.js:83]: Instead of doing this in every block, it may be nice to grab it once in the `before` hook?

**Review by JustinBeckwith — APPROVED** [review]
Generally LGTM.  Please rename this to `docs: add update samples` though!  When you use `feat: ` it's a signal to our release tooling that this change includes material updates to the library, which should lead to a release.

File: samples/test/update_cluster_test.js (hunk 2 above)
Comment (irataxy) [reply samples/test/update_cluster_test.js]: Good idea. Done.

**Comment by tmatsuo** [comment]
The commit message still has `feat:` I'm not sure if our release tool refers to the PR title v.s. to the commit message. Do you know @JustinBeckwith ?

**Pull request merged**